README.md

```diff
@@ -1,13 +1,14 @@
+<img src="https://sajjaadf.github.io/chromebrew-Dev/brew.png" alt="CBlogo" width="123.5" height="145.75">
+
 chromebrew
 ==========
 
 Package manager for Chrome OS
 
 Chat with us!
 -------------
-Messages are synced between services for your convenience. 😊
-
-<a href="https://join.slack.com/t/chromebrew/shared_invite/enQtNDA2MTQ5ODQ3NDc2LTA0ZmJlMGFmNmZhOTYwNDE3ZDY0NDA0MWI0OTE3MzJkYTQxN2UxMWQ1YTEzOWFmNTliMGM4NDZjYzY2NjczZGE" target="_blank"><img src="/images/slack.png" alt="Slack Invite" /></a><a href="https://discord.gg/QRrzBXN" target="_blank"><img src="/images/discord.png" alt="Discord Invite" /></a>
+<I><pre>Discord is Down and not syncing, unfortunately</pre></I>
```

Comment:
`<p><em>...</em></p>` would be better than `<I><pre>...</pre></I>`.  Also, what happened to the Discord icon and the better message?  `Discord is not currently syncing messages with Slack`.  Discord is not down, it's just not syncing messages with Slack.